app/src/main/java/com/droidhats/campuscompass/repositories/NavigationRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.droidhats.campuscompass.repositories
```

Comment:
It's cool that you took the initiative to start off this repository, however, the functionality in this file will be refactored out soon, as this class will become an abstract class for the indoor and outdoor navigation